src/libs/memoize/const.ts

```diff
@@ -0,0 +1,11 @@
+import type {Options} from './types';
+
+const DEFAULT_OPTIONS = {
+    maxSize: Infinity,
+    equality: 'deep',
+    monitor: false,
+    cache: 'array',
+} satisfies Options;
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Done ✅ 

However, with nature of such files where it is intended to export separate constants (different configs, options etc.) it would result in having to fix all the previous imports instead of removing the `eslint-disable` line internally. 

tl;dr; Adding any additional import to this module will break others in the future. WDYT?

Reply:
>  it would result in having to fix all the previous imports

I don't follow ... how are these any different?

```ts
// w/ default export
export default {
   a: 1,
   b: 2,
   c: 3,
};
import {a, b} from './myModule';

// w/ named export
export {
    a: 1,
    b: 2,
    c: 3,
};
import {a, b} from './myModule';
```

Reply:
Here is a little demo: https://replit.com/@kacpermikolajc1/ts-export-stuff#index.ts

Let me know if this address your concerns 🙏

Reply:
oh, yeah my mistake I thought it was possible to destructure a default export when importing it.

Reply:
No worries, I find `import` and `export` syntax quite tricky - it looks straightforward but hides some cool gotcha moments. 

Sharing a link to article from REPL here as well, for someone who is going to stumble upon the PR: https://jakearchibald.com/2021/export-default-thing-vs-thing-as-default/